pages/contact.html

```diff
@@ -65,24 +66,72 @@
+    <p>Email School: <a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a></p>
+    <p>Web: mountsioncbssecondary.ie</p>
+    <p>Telephone Number: 051 377378</p>
+    <p>Fax Number: 051-376468</p>
+    <p>School Roll No. 64930i</p>
+    <p>Address:</p>
+    <p>Mount Sion CBS Secondary School,</p>
+    <p>Barrack Street,</p>
+    <p>Waterford City.</p>
+    <h1 class="ui dividing header" style="padding-top:8px">School Location</h1>
+    <p>School Location on Google maps</p>
+    <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d9769.151915664062!2d-7.1166274!3d52.2563154!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xf8355ef4efe9f1a4!2sMount+Sion+CBS+Secondary+School!5e0!3m2!1sen!2sie!4v1523822538739" width="100%" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+</div>
+<br/>
+
+<div class="ui inverted blue vertical footer segment">
+    <div class="ui center aligned container">
+      <div class="ui inverted section divider"></div>
+        <div class="ui stackable divided grid">
+            <div class="four wide column">
+                <div class="ui link list">
+                    <a href="https://mountsion.vsware.ie/Login.jsp" class="item">
+                        Vs Ware
+                    </a>
+                    <a href="mailto:info@mountsioncbssecondary.ie" target="_top" class="item">
+                        Email
+                    </a>
+                    <a href="https://www.education.ie/en/" class="item">
+                        Department of Education
+                    </a>
+                    <a href="https://www.examinations.ie" class="item">
+                        State Exams Commission
+                    </a>
+                </div>
+            </div>
+            <div class="four wide column">
+                <div class="ui link list">
+                    <a href="http://www.pdst.ie" class="item">
+                        P.D.S.T
+                    </a>
+                    <a href="http://www.ncca.ie" class="item">
+                        N.C.C.A
+                    </a>
+                    <a href="http://www.ncca.ie/en/junior-cycle" class="item">
+                        NCCA Junior Cycle
+                    </a>
+                    <a href="http://www.ncca.ie/en/senior-cycle" class="item">
+                        NCCA Senior Cycle
+                    </a>
+                </div>
+            </div>
+            <div class="eight wide column">
+                <div class="ui link list">
+                <h4 class="ui header">Mount Sion C.B.S Waterford</h4>
+                <p>An all boys school located in the heart of Waterford City</p>
+                <p>It is the first ever school set up by Edmund Rice.</p>
+            </div>
+            </div>
+        </div>
+        <div class="ui section divider"></div>
+        <img class="ui centered small image" src="../assets/images/logo.jpg">    
```

Comment:
As requested in the previous PR either remove this logo or add a crest with no background. 
IMO having the school crest in the footer is a nice touch. Just with the white background, it does not look good removing the background would be a nice touch.